wiki/PhyloDBSchema.md

```diff
@@ -82,7 +82,7 @@ part of. "left\_idx and "right\_idx" specify the left and right values
 of the nested set optimization structure for efficient hierarchical
 queries. These values needs to be precomputed by a program, see [J.
 Celko, SQL for
-Smarties](http://rs18.rapidshare.com/files/4441805/Trees_and_Hierarchies_in_SQL_for_Smarties.pdf).
+Smarties](https://datubaze.files.wordpress.com/2016/03/celkos-trees-and-hierarchies-in-sql_2004.pdf).
```

Comment:
I agree, the PDF is probably unintentionally public, and is fragile as it's subject to a DMCA takedown notice from the copyright holder. I'd suggest to use the ISBN link, here's the one through Worldcat: https://www.worldcat.org/search?q=isbn%3A0128007613